inc/class-sitemaps-index.php

```diff
@@ -44,7 +52,7 @@ public function redirect_canonical( $redirect ) {
 	/**
 	 * Produce XML to output.
 	 *
-	 * @return string
+	 * @return void
```

Comment:
We can completely remove the `return` value here.

Reply:
No, wee need to return $template if the sitemap get variable isn't posts, otherwise the rest of the site goes blank.